norns/maiden.md

```diff
@@ -70,38 +70,49 @@ Each entry has an **INSTALL** action, which can be used to install the selected
 You will have to restart norns if a freshly-installed project contains an engine
 {: .label}
 
-### contribute
+### contribute & gather
 
-To add to the [community project repo](https://github.com/monome/norns-community/blob/master/community.json), please submit a pull request with the following information:
+The norns ecosystem was created with community as a focus. The exchange of ideas leads to new ideas.
 
-```json
-    {
-      "project_name": "NAME",
-      "project_url": "URL",
-      "author": "NAME",
-      "description": "WORDS",
-      "discussion_url": "LINES_LINK",
-      "tags": ["TAG", "TAG", "TAG"],
-      "origin": "IF_APPLICABLE: lines"
-    },
-```
+As such, you are encourage to share your creations with the community.
+
+The first step is to host the sources of your script publicly, either as a git repository (e.g. on GitHub, GitLab ...) or web-hosted zip archive.
 
-Be sure you include information at the top of your script to help future users:
+Be sure you include a block of information at the top of your script. It will serve as a brief documentation when launching it from norns.
 
 ```lua
--- scriptname: short script description
+-- scriptname
 -- v1.0.0 @author
 -- llllllll.co/t/22222
+--
+-- short script description
+--
+-- short usage instructions
 ```
 
-That last line is a link back to the thread number. There's a chicken-egg situation with starting a thread and uploading the project, so you may want to edit and upload your project just after creating a thread.
+Then, create a new thread under the [norns Library on lines](https://llllllll.co/c/library).
 
-### gather
+The URL in your script brief documentation should point to the lines Library thread for your script. There's a chicken-egg situation with starting a thread and uploading the project, so you may want to edit and upload your project just after creating a thread.
+{: .label}
 
-While many projects are held in the community repository, it can be hard to engage with a project's creator through GitHub. To facilitate discussion, many projects are also shared through the [norns Library on lines](https://llllllll.co/c/library).
+You can optionally declare it on the wiki ([norns.community](/docs/norns/community-scripts/)) and write a small documentation about its usage.
+
+So that anyone can do a 1-click install using maiden maiden, it is encourage to register your script to the [community project repo](https://github.com/monome/norns-community/blob/main/community.json). Please submit a pull request with the following information:
```

Comment:
rephrase suggestion:

> So that anyone can do a one-click install using maiden, we encourage you to register your script in the [community project repo](https://github.com/monome/norns-community/blob/main/community.json). Please submit a pull request with the following information: